sql/executor.go

```diff
@@ -283,8 +284,12 @@ func (p parameters) Arg(name string) (parser.Datum, bool) {
 		return parser.DBytes(t.BytesVal), true
 	case *driver.Datum_StringVal:
 		return parser.DString(t.StringVal), true
+	case *driver.Datum_DateVal:
+		return parser.DTimestamp{Time: t.DateVal.GoTime().UTC()}, true
```

Comment:
Per Ben's comment in another change, should this be `GoUTCTime()` or something like that?